### DISCUSSION TITLE
toString() output format

### DISCUSSION THREAD

**Discussion by nthomas99** [josdejong/mathjs#2176] [Q&A] [answered]
Howdy All,

Mathjs is awesome sauce.  I'm using it to read in an asciimath specification, build the parse tree, do some transformations, and emit latex for rendering in a scientific app.  I also want to send this this to microsoft word (using another package), which is expecting OMML (which for the basic stuff I'm using I think I can create for MathML).  

What exactly is the text format toString() emits?  Is it any specific standard? 

Thanks!

**Comment by josdejong** [comment, marked as answer]
The output of toString is the same format as the input you're feeding to the expression parser. It's documented in detail here: https://mathjs.org/docs/expressions/syntax.html
